### ISSUE TITLE
Initialisms are missed when used in mixed-case pluralization

### ISSUE THREAD

**Issue by displague** [go-swagger/go-swagger#2522]
## Problem statement

When specifying Initialisms, these initialisms should be aware of mixed-case used in plurals:

`VLANs`, `IPs`, should be treated as `vlans`, `ips`. 

What is generated today is `v_l_a_ns`

## Swagger specification

https://github.com/t0mk/gometal/blob/main/equinix-metal.patched.json

## Steps to reproduce

https://github.com/t0mk/gometal/blob/main/Makefile#L29-L35

https://github.com/t0mk/gometal/tree/main/client/v_l_a_ns

## Environment
swagger version: 2.0
go version: 0.26.1

**Comment by fredbi** [comment]
Similar to https://github.com/go-openapi/swag/issues/46

**Comment by fredbi** [comment]
As a workaround, you can add you pluralized forms as extra initialisms to the command line  options.

**Comment by gregmarr** [comment]
The function for adding more initialisms converts the provided string to uppercase, so that won't help here.

**Comment by displague** [comment]
Indeed. I have `vlan` and `vlans` in my initialialisms lists (see the Makefile link above) and the results in a `v_l_a_ns` client directory.
